fairlearn/metrics/_generated_metrics.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import sklearn.metrics as skm
+
+from ._extra_metrics import (
+    true_positive_rate, true_negative_rate,
+    false_positive_rate, false_negative_rate,
+    selection_rate,
+)
+from ._make_derived_metric import make_derived_metric
```

Comment:
can't this be `from . import make_derived_metric`?

Reply:
Apparently so. I was concerned that that would end up recursing on something not fully formed.